deployment/upgradeLive

```diff
@@ -6,30 +6,37 @@ set -e
 cd /opt/kaltura/liveController
 echo updating liveController v$1
 if [ ! -d "$1" ] ; then
-	if [ -L "${latest}" ]; then
-		echo "unlink ${latest}"
-		unlink ${latest}
-	fi
     echo Try to download  https://github.com/kaltura/liveDVR/archive/v$1.tar.gz
     wget https://github.com/kaltura/liveDVR/archive/v$1.tar.gz
     echo try to unzip v$1.tar.gz
     tar -xvzf v$1.tar.gz
     mv liveDVR-$1 $1
+    rm -rf /opt/kaltura/liveController/v$1.tar.gz
     cd $1
+    echo "current dir: [`pwd`]"
     nvm install
     # move configuration files (liveController's and liveRecorder's) to latest version
     cp /opt/kaltura/liveController/configMapping.json /opt/kaltura/liveController/$1/common/config/
     cp /opt/kaltura/liveController/configMapping.ini /opt/kaltura/liveController/$1/liveRecorder/Config/
     npm install
     cd ..
     # download binaries from Jenkins
+    echo "### starting to download binaries"
     bash /opt/kaltura/liveController/$1/deployment/get_bins "$1"
-    rm -rf /opt/kaltura/liveController/v$1.tar.gz
-    # create symlinks to latest dir and process script
-    ln -s /opt/kaltura/liveController/$1 /opt/kaltura/liveController/latest
-    ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController
+    echo "### finished downloading binaries"
     # start nginx (web server), liveController and liveRecorder
     /opt/kaltura/liveController/$1/packager/bin/run_nginx.sh
+    # create symlinks to latest dir and process script
+	if [ -L "latest" ]; then
+		echo "unlink latest"
+		unlink latest
+	fi
+	if [ -L /etc/init.d/kLiveController ]; then
+		echo "unlink /etc/init.d/kLiveController"
+		unlink /etc/init.d/kLiveController
+	fi
```

Comment:
Don't understand the need to unlink the kLiveController script, as creating it again overrides it